Decompiler/Decompiler.cs

```diff
@@ -865,29 +865,21 @@ private void DumpVPK(string parentPath, Package package, string type, Dictionary
 
                         extension = FileExtract.GetExtension(resource) ?? type[..^2];
 
-                        // TODO: Hook this up in FileExtract
-                        if (resource.ResourceType == ResourceType.Mesh || resource.ResourceType == ResourceType.Model)
+                        if (GltfModelExporter.CanExport(resource))
```

Comment:
Missed by #344 or is it more helpful that Decompiler only does models?

Reply:
Just missed.